repytah/utilities.py

```diff
@@ -212,83 +206,63 @@ def find_initial_repeats(thresh_mat, bandwidth_vec, thresh_bw):
                     # NOTE: the middle interval may NOT exist
                     # Vector of 1's that is the length of the number of
                     # overlapping intervals. This is used a lot.
-                    ones_no = np.ones(num_overlaps)
+                    ones_no = np.ones(num_overlaps, int)
```

Comment:
I'm a little bit confused why sometimes we use `ones_no` and sometimes we can just use scalar 1? 
Ex. Line 235 uses 1 in `ematch_i_over = end_i_over - K + 1` but Line 257 uses `ones_no` in `end_i_over - start_j_over - K + ones_no`. Both are using it on the same variable.

Reply:
Good point. I'll remove the ones_no